lib/icon.js

```diff
@@ -0,0 +1,305 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const IconBase = styled.img`
+  display: inline-block;
+  vertical-align: top;
+  height: 0.9em;
```

Comment:
I think everything looks great but I have some thoughts on the way sizing is done. This component defines its height as an `em` value and depends on its context's `font-size` to size itself. It can only be sized via `font-size`. If I wanted to render a large icon in an onboarding tip, like this:

<img width="299" alt="Screen Shot 2019-07-26 at 10 11 40 AM" src="https://user-images.githubusercontent.com/654275/61957673-c90f8d80-af8d-11e9-8d5b-b656ded72d11.png">

Wouldn't I need to wrap it in a div where the `font-size` was set to something like `48px`? That seems too indirect to me, especially in cases where we're not rendering an icon inline with text.

I'd prefer being able to pass in a named sized, like this:

```
<Icon size="medium" />
```

Where we have a defined set of sizes corresponding to pixel values. And maybe there's a special value like `inline` where the icon's height is set to `0.9em`.

Reply:
sure -- I imagine you'll have a similar comment on the Loader component.